test/superv/async_test.cljc

```diff
@@ -335,92 +341,92 @@
                   42))))
      (let [start-fn (fn [S]
                       (go-super S
-                        (throw (ex-info "foo" {}))))]
-       (go (is (thrown? Exception
+                                (throw (ex-info "foo" {}))))]
+       (go (is (thrown? #?(:clj Exception :cljs js/Error)
                         (<? S (restarting-supervisor start-fn :retries 3 :stale-timeout 100))))))
+
+        ;#?(:clj)  ;; TODO: get this one working in cljs
```

Comment:
The CI issue comes from here @TimoKramer @whilo 
Seems this issue was not fixed as the TODO comment is still here.